app/controllers/api/v2/anime_controller.rb

```diff
@@ -2,6 +2,21 @@ module Api::V2
   class AnimeController < ApiController
     caches_action :show, expires_in: 1.hour
 
+    def index
+      if params.key? :filter
+        search = AnimeSearch.new params[:filter]
```

Comment:
Methods which aren't part of a DSL should generally have parentheses for their call (this is a vague distinction, I know)